lte/gateway/python/integ_tests/s1aptests/test_scalability_attach_detach_multi_ue.py

```diff
@@ -0,0 +1,95 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+import unittest
+
+import s1ap_types
+import s1ap_wrapper
+
+
+class TestScalabilityAttachDetachMultiUe(unittest.TestCase):
+    """Integration Test: TestScalabilityAttachDetachMultiUe"""
+
+    def setUp(self):
+        """Initialize before test case execution"""
+        self.default_ip_block = s1ap_wrapper.TestWrapper.TEST_IP_BLOCK
+        s1ap_wrapper.TestWrapper.TEST_IP_BLOCK = "192.168.128.0/17"
```

Comment:
In case if someone wants to execute this test case for different number of UEs, can we add comment to inform that mask value needs to be changed with logic indicated in s1ap_wrapper.py file or same comment can be added here as well

Reply:
Added comment with reference to s1ap_wrapper.py as well as S1APTester documentation to get information from whichever is updated latest.